pkg/metrics/metrics.go

```diff
@@ -117,14 +117,16 @@ func trackMetrics(app *application.Avalanche, flags map[string]string, cmdErr er
 
 	telemetryProperties := make(map[string]interface{})
 	telemetryProperties["command"] = app.Cmd.CommandPath()
-	telemetryProperties["version"] = version
+	telemetryProperties["cli_version"] = version
 	telemetryProperties["os"] = runtime.GOOS
-	telemetryProperties["error"] = ""
+	telemetryProperties["was_successful"] = true
+	telemetryProperties["error_msg"] = ""
 	if cmdErr != nil {
-		telemetryProperties["error"] = cmdErr.Error()
+		telemetryProperties["was_successful"] = false
+		telemetryProperties["error_msg"] = cmdErr.Error()
```

Comment:
why can't we just check if error_msg is empty to check if there is error

Reply:
meaning there is no need for was_successful

Reply:
I see. It is for convenience in using the metrics analytics tool, that works better with simple boolean on basic dashboards